airflow/config_templates/config.yml

```diff
@@ -1303,6 +1303,13 @@
       type: string
       example: ~
       default:
+    - name: instance_name_markup
```

Comment:
Since it is a boolean value, I would suggest a name that makes it seems so.

Few suggestions: `has_markup_in_instance_name`, `markup_in_instance_name`

Reply:
Or `instance_name_is_html`